### ISSUE TITLE
panic in git scanner

### ISSUE THREAD

**Issue by smola** [src-d/lookout#304]
Executing on gitbase-web repository:

```
$ lookout-sdk review -v ipv4://localhost:2000 --from=HEAD^^^^^^^^^^^  --to=HEAD
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xa9eda6]

goroutine 75 [running]:
github.com/src-d/lookout/service/git.(*regexpFilter).Fn(0xc420c7b4a0, 0x0, 0x0, 0x0, 0xc420c8ed80)
        /home/travis/gopath/src/github.com/src-d/lookout/service/git/scanner.go:172 +0x26
github.com/src-d/lookout/service/git.NewChangeFilterScanner.func1(0xc420aaafa0, 0xc420aaafa0, 0xc420db76d0, 0xc420bc9a20)
        /home/travis/gopath/src/github.com/src-d/lookout/service/git/scanner.go:215 +0x38
github.com/src-d/lookout.(*FnChangeScanner).Next(0xc42142d500, 0xc420e8c000)
        /home/travis/gopath/src/github.com/src-d/lookout/data.go:288 +0x99
github.com/src-d/lookout.(*FnChangeScanner).Next(0xc42142d540, 0x0)
        /home/travis/gopath/src/github.com/src-d/lookout/data.go:286 +0x5a
github.com/src-d/lookout.(*FnChangeScanner).Next(0xc42142d580, 0xc420205300)
        /home/travis/gopath/src/github.com/src-d/lookout/data.go:286 +0x5a
github.com/src-d/lookout.(*FnChangeScanner).Next(0xc42142d5c0, 0xc420205300)
        /home/travis/gopath/src/github.com/src-d/lookout/data.go:286 +0x5a
github.com/src-d/lookout/service/bblfsh.(*ChangeScanner).Next(0xc420246a00, 0xc420205330)
        /home/travis/gopath/src/github.com/src-d/lookout/service/bblfsh/bblfsh.go:141 +0x45
github.com/src-d/lookout.(*DataServerHandler).GetChanges(0xc420c0d1e0, 0xc42142d140, 0xf77720, 0xc421002ab0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/src-d/lookout/data.go:86 +0x151
github.com/src-d/lookout/pb._Data_GetChanges_Handler(0xc39dc0, 0xc420c0d1e0, 0xf76700, 0xc421311a40, 0x0, 0xc4214f5f90)
        /home/travis/gopath/src/github.com/src-d/lookout/pb/service_data.pb.go:194 +0x10e
github.com/src-d/lookout/util/grpchelper.StreamServerInterceptor.func1(0xc39dc0, 0xc420c0d1e0, 0xf76820, 0xc4200d7130, 0xc421311980, 0xe5c820, 0xc42138002d, 0xc4200674c0)
        /home/travis/gopath/src/github.com/src-d/lookout/util/grpchelper/logger.go:47 +0x153
github.com/src-d/lookout/vendor/google.golang.org/grpc.(*Server).processStreamingRPC(0xc4200ec380, 0xf78aa0, 0xc420a42200, 0xc4201ec200, 0xc4204ad8c0, 0x173c5e0, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/src-d/lookout/vendor/google.golang.org/grpc/server.go:1167 +0x496
github.com/src-d/lookout/vendor/google.golang.org/grpc.(*Server).handleStream(0xc4200ec380, 0xf78aa0, 0xc420a42200, 0xc4201ec200, 0x0)
        /home/travis/gopath/src/github.com/src-d/lookout/vendor/google.golang.org/grpc/server.go:1253 +0x12b1
github.com/src-d/lookout/vendor/google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc4214ef3a0, 0xc4200ec380, 0xf78aa0, 0xc420a42200, 0xc4201ec200)
        /home/travis/gopath/src/github.com/src-d/lookout/vendor/google.golang.org/grpc/server.go:680 +0x9f
created by github.com/src-d/lookout/vendor/google.golang.org/grpc.(*Server).serveStreams.func1
        /home/travis/gopath/src/github.com/src-d/lookout/vendor/google.golang.org/grpc/server.go:678 +0xa1
```

**Comment by smola** [comment]
Using a hash instead of `HEAD^^^^^^^^^^^` works.

**Comment by carlosms** [comment]
I'm pretty sure this is bug #261. Can you please confirm your `lookout-sdk` version? The fix was included in v0.1.5.

**Comment by smola** [comment]
```
$ lookout-sdk version
lookout-sdk v0.1.4 built on 09-04-2018_06_15_05
```

@carlosms  Ok. Freel free to close.
